Simplenote/SPFallbackTextContainer.swift

```diff
@@ -0,0 +1,19 @@
+class SPFallbackTextContainer: NSTextContainer {
+    weak var textView: SPTextView?
+
+    override var layoutManager: NSLayoutManager? {
+        didSet {
+            // This gets called when falling back to TextKit 1
+            guard let layoutManager,
+                  let newManagerStorage = layoutManager.textStorage,
+                  let textView,
+                  textView.interactiveTextStorage.layoutManagers.isEmpty else {
+                return
+            }
+
+            textView.interactiveTextStorage.addLayoutManager(layoutManager)
+            textView.interactiveTextStorage.setAttributedString(newManagerStorage)
+            textView.text = newManagerStorage.string
```

Comment:
Can we move this so that it's explicit during the initialization flow, rather than a lazy thing?